api/src/pcapi/local_providers/titelive_things/titelive_things.py

```diff
@@ -95,6 +96,9 @@ def __init__(self):  # type: ignore [no-untyped-def]
         self.data_lines = None
         self.products_file = None
         self.product_extra_data = offers_models.OfferExtraData()
+        self.product_whitelist_eans = [
+            ean for ean, in fraud_models.ProductWhitelist.query.with_entities(fraud_models.ProductWhitelist.ean).all()
+        ]
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Il me semble que `with_entities` retourne un tuple, même avec un seul élément
Du coup ici il faudrait `ean[0] for ean in ...` Peut être renommer la variable `ean` en `result` ou `ean_tuple` ou autre

Reply:
Je convertis le tuple avec la boucle for, du coup c'est un array de ean

Reply:
Aah c'est la virgule qui fait ça, nice :+1: